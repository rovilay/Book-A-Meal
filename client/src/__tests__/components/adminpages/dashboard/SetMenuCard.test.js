import React from 'react';
import { shallow } from 'enzyme';
import SetMenuCard from '../../../../components/adminpages/dashboard/MenuCard/SetMenuCard';

const props = {
  editMenu: jest.fn(),
  getMenuMeals: jest.fn(),
  submitNewMenu: jest.fn()
}

describe('SetMenuCard component test', () => {
  const setup = () => shallow(<SetMenuCard {...props}/>);


  it('should render SetMenuCard correctly', (done) => {
    const wrapper = setup();

    expect(wrapper).toMatchSnapshot();

    done();
  });
});
