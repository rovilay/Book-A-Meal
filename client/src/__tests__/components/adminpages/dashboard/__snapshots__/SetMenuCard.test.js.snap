// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SetMenuCard component test should render SetMenuCard correctly 1`] = `
<AccordionItemWrapper
  accordionStore={
    AccordionContainer {
      "_listeners": Array [],
      "addItem": [Function],
      "removeItem": [Function],
      "setAccordion": [Function],
      "setExpanded": [Function],
      "setOnChange": [Function],
      "state": Object {
        "accordion": true,
        "items": Array [],
        "onChange": [Function],
      },
    }
  }
  className="accordion__item"
  disabled={false}
  expanded={false}
  hideBodyClassName=""
>
  <AccordionItemTitleWrapper
    className="accordion__title"
    hideBodyClassName=""
  >
    <h3>
      <span
        className="postOn"
      >
        Create Menu
      </span>
    </h3>
    <FontAwesome
      name="chevron-down"
    />
  </AccordionItemTitleWrapper>
  <AccordionItemBodyWrapper
    className="accordion__body"
    hideBodyClassName="accordion__body--hidden"
  >
    <div
      className="setMenuCard"
    >
      <hr />
      <SetMenuForm
        editMenu={[MockFunction]}
        getMenuMeals={[MockFunction]}
        submitNewMenu={[MockFunction]}
      />
    </div>
  </AccordionItemBodyWrapper>
</AccordionItemWrapper>
`;
